pages/calculator/[slug].js

```diff
@@ -41,6 +49,9 @@ export default function CalculatorSlugRoute({ page, calculatorConfig }) {
 
   const isFirstPage = () => page.slug === 'head-initial-1-cont';
 
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('sm'));
```

Comment:
All of the other components that do this check call the variable `matchesXS`, let's keep them all the same.

Reply:
Eleni I am looking for this matchesXS variable in the slug.js page and I simply can't find it. Could you direct me to where it may be

Reply:
Ah sorry, I mean in other files the isMobile var is named matchesXS, so here we should just rename it.

Reply:
Renamed.